api/controllers/ResumeController.ts

```diff
@@ -65,4 +67,11 @@ export class ResumeController {
     const resumes = await this.resumeService.getVisibleResumes();
     return { error: null, resumes: resumes.map((resume) => resume.getPublicResume()) };
   }
+
+  @Delete()
+  async deleteResume(@Params() params: UuidParam, @AuthenticatedUser() user: UserModel): Promise<DeleteResumeResponse> {
+    const resume = await this.resumeService.deleteResume(params.uuid, user);
+    if (resume) await this.storageService.deleteAtUrl(resume.url);
+    return { error: null };
+  }
```

Comment:
Do we need this `if (resume)` check? When would there be a case where `resume` is falsy? It seems like we're already throwing an error if resume is falsy in the service function, so to me it doesn't seem like `resume` would ever be falsy on line 74.

Reply:
You are right, this check was not necessary.